pkg/epp/requestcontrol/director.go

```diff
@@ -158,12 +155,15 @@ func (d *Director) HandleRequest(ctx context.Context, reqCtx *handlers.RequestCo
 
 // admitRequest handles admission control to decide whether or not to accept the request
 // based on the request criticality and system saturation state.
-func (d *Director) admitRequest(ctx context.Context, requestCriticality v1alpha2.Criticality, fairnessID string) error {
+func (d *Director) admitRequest(ctx context.Context, requestCriticality int, fairnessID string) error {
 	logger := log.FromContext(ctx)
 
 	logger.V(logutil.TRACE).Info("Entering Flow Control", "criticality", requestCriticality, "fairnessID", fairnessID)
 
-	if requestCriticality == v1alpha2.Critical {
+	// This will be removed in favor of a more robust implementation (Flow Control) in the very near future.
+	// For now we will keep similar behavior to the previous implementation.
+	// TODO: Make this a configurable value.
```

Comment:
yup. ok for now to keep same behavior.
should be documented somewhere though (it might be surprising)